src/app/login/login.component.html

```diff
@@ -0,0 +1,25 @@
+<div class='login-body'>
+    <div class='login-card'>
+        <div class="image-area">
+            <img src="assets/tavisca-logo.png" class="" alt="Tavisca">
+        </div>
+
+        <div class="container">
+            <div class='card-message'>
+                <p>Sign In</p>
+            </div>
+            <!-- Default input -->
+            <div class="inputs">
+                <input type="text" class="form-control" placeholder="Enter Username" [(ngModel)]="userDetails.Username">
```

Comment:
Have you handled any validations for username and pwd here. Might need a discussion on if its required